src/main/java/jenkins/plugins/office365connector/FactsBuilder.java

```diff
@@ -109,10 +109,7 @@ public void addCulprits(Set<User> authors) {
             return;
         }
 
-        Set<String> culprits = new HashSet<>();
-        for (User user : authors) {
-            culprits.add(user.getFullName());
-        }
+        List<String> culprits = authors.stream().map(user -> user.getFullName()).collect(Collectors.toList());
```

Comment:
Use method reference
```suggestion
        List<String> culprits = authors.stream().map(User::getFullName).collect(Collectors.toList());
```